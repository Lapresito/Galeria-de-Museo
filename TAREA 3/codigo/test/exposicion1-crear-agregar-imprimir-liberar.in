crearExposicion 01/01/2001 01/06/2001 10 1
imprimirExposicion
liberarExposicion
crearExposicion 01/01/2002 01/06/2002 10 1
agregarAExposicion 5 0 1 2 3 4
imprimirExposicion
agregarAExposicion 5 5 6 7 8 9
imprimirExposicion
agregarAExposicion 5 10 11 12 13 14
imprimirExposicion
liberarExposicion
Fin