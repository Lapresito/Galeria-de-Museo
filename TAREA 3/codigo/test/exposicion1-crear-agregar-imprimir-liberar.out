Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Exposicion creada con exito.
2> Exposicion #1 del 1/1/2001 al 1/6/2001
Piezas: 
3> Exposicion liberada con exito.
4> Exposicion creada con exito.
5> Piezas agregadas a la exposicion de forma exitosa.
6> Exposicion #1 del 1/1/2002 al 1/6/2002
Piezas: 0 1 2 3 4 
7> Piezas agregadas a la exposicion de forma exitosa.
8> Exposicion #1 del 1/1/2002 al 1/6/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
9> Piezas agregadas a la exposicion de forma exitosa.
10> Exposicion #1 del 1/1/2002 al 1/6/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
11> Exposicion liberada con exito.
12> Fin.
